start.sh

```
@@ -1,28 +1,44 @@
+    docker rm ${CONTAINER_NAME}
+fi
 
-# 启动容器，挂载当前目录到 /app
+# 启动容器，挂载工作目录但保护登录服务器的依赖
 docker run -d \
-    --name cloud-code-dev \
-    -p 7681:7681 \
-    -v $(pwd):/app \
+    --name ${CONTAINER_NAME} \
+    -p 80:80 \
+    -v $(pwd):/workspace \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)